doc/Makefile

```diff
@@ -26,5 +28,61 @@ help:
         "* other possible targets:                    make <TAB twice> \n" \
         "------------------------------------------------------------- \n"
 
+integrate:
+	# Pull the other repositories
+	cd integration/ && \
+	git -C lxd fetch && git -C lxd reset --hard origin/main || git clone https://github.com/canonical/lxd
```

Comment:
As discussed we should pull a version that matches our compatibility matrix (once that gets added), as it may not be immediately obvious to a user that they haven't been redirected to https://documentation.ubuntu.com/lxd.

Not clear yet how we will specify compatible versions in the `microcloud` executable (RE version enforcement), but it would be ideal if those versions were used here to determine the correct branch. Once again, not a blocker.

Could you open an issue to track this concern with any additional context?

Reply:
I opened https://github.com/canonical/microcloud/issues/374, and updated the Makefile to specify the versions at the top.